main.go

```diff
@@ -85,15 +85,16 @@ func main() {
 		os.Exit(1)
 	}
 
-	if namespace, found := os.LookupEnv("OPERATOR_NAMESPACE"); found {
```

Comment:
I think that we can remove the [env var entirely](https://github.com/medik8s/node-maintenance-operator/blob/main/config/default/manager_namespace_patch.yaml) if we remove it from here (the single place it has been used) .

Reply:
Hmm, I don't follow.
The line you've marked (88) is  already removed as part of the PR, is there any leftover you refer to ?

Reply:
> is there any leftover you refer to ?

In line 88 we are fetching the **OPERATOR_NAMESPACE** env var, and if we remove this line, then we don't use this env var anymore in the code. It becomes redundant.
I was suggesting that if we want to remove line 88, then it makes sense to remove the env var entirely.
It can be done by commenting in [kustomization.yaml](https://github.com/medik8s/node-maintenance-operator/blob/main/config/default/kustomization.yaml#L44) the _manager_namespace_patch.yaml_ file.

Reply:
As discussed, this env var is not part of this PR so we may address this issue in a different PR .